src/layouts/ReviewRequest/Dashboard.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  HStack,
+  VStack,
+  Text,
+  Box,
+  Avatar,
+  Flex,
+  Spacer,
+  useClipboard,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  PopoverArrow,
+} from "@chakra-ui/react"
+import { IconButton } from "@opengovsg/design-system-react"
+import {
+  MenuDropdownButton,
+  MenuDropdownItem,
+} from "components/MenuDropdownButton"
+import { useState } from "react"
+import { BiLink } from "react-icons/bi"
+
+import { extractInitials, getDateTimeFromUnixTime } from "utils"
+
+import { RequestOverview, EditedItemProps } from "./components/RequestOverview"
+
+export interface ReviewRequestDashboardProps {
+  siteName: string
+  reviewUrl: string
+  title: string
+  requestor: string
+  reviewers: string[]
+  changedItems: EditedItemProps[]
+}
+export const ReviewRequestDashboard = ({
+  siteName,
+  reviewUrl,
+  title,
+  requestor,
+  reviewers,
+  changedItems,
+}: ReviewRequestDashboardProps): JSX.Element => {
+  const { onCopy, hasCopied } = useClipboard(reviewUrl)
+
+  return (
+    <Box bg="white" w="100%" h="100%">
+      <VStack
+        bg="blue.50"
+        pl="9.25rem"
+        pr="2rem"
```

Comment:
Would it work to have these 2 values moved to the `<Box>` above? So then the same `pl` and `pr` values for the RequestOverview Box can be removed.

Reply:
sadly we can't cos this results in the top having white bg... an alternatiev would be to have the `Box` have the blue bg then we set the `Box` wrapping the `RequestOverview` to have white bg - would you prefer this? (then we shift the `pl + pr` to above)

Reply:
Oh I think it's fine, let's just stick to the current way of doing it!